config/backend-config/workspace-config.go

```diff
@@ -67,8 +101,9 @@ func (workspaceConfig *WorkspaceConfig) getFromAPI() (SourcesT, bool) {
 		return SourcesT{}, false
 	}
 
+	updatedResponseBody := replaceConfigWithEnvVariables(respBody)
 	var sourcesJSON SourcesT
-	err = json.Unmarshal(respBody, &sourcesJSON)
+	err = json.Unmarshal(updatedResponseBody, &sourcesJSON)
```

Comment:
Can we rename the function from getFromApi or move this logic to some other function?